tests/assets/repository/index.yaml

```diff
@@ -0,0 +1,60 @@
+{
+  "generated": "2020-05-06T20:30:58.581950",
+  "operator": {
+    "honeybee-radiance": [
+      {
+        "name": "honeybee-radiance",
+        "version": "1.2.3",
+        "appVersion": "2.3.2",
```

Comment:
`app-version` ?

Reply:
Picked it up from helm. The Operator version corresponds to the version of the YAML file + version of container or library. You might add a new function but not change the version of the underlying package/container. In which case you indicate what the "appVersion".

Reply:
Thanks. My comment was mainly about the semantics. I think we should be consistent on keywords and not use camelCase here! 😁

Reply:
Oh! Hadn't thought about it in that way. I'll create an issue to tackle this in a coordinated manner as there are probably a couple more lying around! https://github.com/ladybug-tools/queenbee/issues/50